ntbs-service/appsettings.Development.json

```diff
@@ -38,7 +39,6 @@
     "TablePrefix": "Dev"
   },
   "ScheduledJobsConfig": {
-    "UserSyncEnabled": false,
```

Comment:
Ah, but I think we still want the `"UserSyncEnabled": false` option

Reply:
Whoops, sorry! Fixed now